### DISCUSSION TITLE
submission keeps "evaluating”

### DISCUSSION THREAD

**Discussion by zlj-cs** [PaddlePaddle/PaddleSpatial#110] [BaiduKDDCup2022] [answered]
my submission keeps "evaluating" for more than six hours and does not show any error or result. Is it still evaluating, or is there any bug in evaluation?
![image](https://user-images.githubusercontent.com/65325751/170527581-dbcdfa65-fa04-47ec-94cc-fe6959cf5af7.png)

**Comment by QiZhou1997** [comment]
I stuck with a similar issue. I submit my code using the 'base' framework but find that 'lightgbm' has not been installed (https://github.com/PaddlePaddle/PaddleSpatial/discussions/102). Therefore, I download lightgbm locally and include it in my submitted code. The local package does fix the " 'NoneType' Object has no attribute 'forecast' " error. However, my submission remains the 'Evaluating' status for more than five hours. I wonder whether the packages in the 'base' environment are installed correctly as I find similar issues have been reported several times. (https://github.com/PaddlePaddle/PaddleSpatial/discussions/97)

**Reply by QiZhou1997** [reply]
> I stuck with a similar issue. I submit my code using the 'base' framework but find that 'lightgbm' has not been installed (#102). Therefore, I download lightgbm locally and include it in my submitted code. The local package does fix the " 'NoneType' Object has no attribute 'forecast' " error. However, my submission remains the 'Evaluating' status for more than five hours. I wonder whether the packages in the 'base' environment are installed correctly as I find similar issues have been reported several times. (#97)

My team name is ‘GoodLuck‘. My last submission is 
<img width="622" alt="image" src="https://user-images.githubusercontent.com/26005993/170535347-e07efa1d-b211-440c-add5-7c9c2a687f9f.png">

**Reply by ramber1836** [reply]
@QiZhou1997 
Your evaluation takes 32626.061465024948 secs (the current online test timeout is 10h), which should be optimized to avoid future timeout errors.

**Comment by ramber1836** [comment]
Hi, there, 

Apologies for the 'lightgbm not found error'. 
We are cooperating with the AIStudio engineer to see what happens in the base environment. 
Many thanks for your understanding.

**Comment by ramber1836** [comment, marked as answer]
The 'lightgbm not found' error has already been fixed for the base env. Thanks.
